testcases/healthcheck/healthcheck.robot

```diff
@@ -0,0 +1,7 @@
+*** Settings ***
+Resource          ./healthcheck_kw.robot
+
+*** Test Cases ***
+Healthcheck
+    [Tags]    other
```

Comment:
Please add **prerelease** tag so that the test will be excluded until your changes lands in PyPI

Reply:
Done.

Reply:
should be **other** (to include in CI) and **prerelease**.